tests/Unit/Domain/DomainCreators/Test_Brick.cpp

```diff
@@ -0,0 +1,111 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "Domain/CoordinateMaps/AffineMap.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/DomainCreators/Brick.hpp"
+#include "Utilities/MakeVector.hpp"
+#include "tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp"
+#include "tests/Unit/Domain/TestDomainHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+using affine_map = CoordinateMaps::AffineMap;
+using affine_map_3d =
+    CoordinateMaps::ProductOf3Maps<affine_map, affine_map, affine_map>;
+void test_brick(const DomainCreators::Brick& brick,
+                const std::array<double, 3>& lower_bound,
+                const std::array<double, 3>& upper_bound,
+                const std::array<size_t, 3>& expected_extents,
+                const std::array<size_t, 3>& expected_refinement_level) {
+  const auto domain = brick.create_domain();
+  const auto& block = domain.blocks()[0];
+  const auto& neighbors = block.neighbors();
+  const auto& external_boundaries = block.external_boundaries();
+
+  CHECK(block.id() == 0);
+  CHECK(brick.initial_extents(0) == expected_extents);
+  CHECK(brick.initial_refinement_levels(0) == expected_refinement_level);
+
+  test_domain_construction(
+      domain,
+      std::vector<std::unordered_map<Direction<3>, BlockNeighbor<3>>>{{}},
+      std::vector<std::unordered_set<Direction<3>>>{
+          {{Direction<3>::lower_xi()},
+           {Direction<3>::upper_xi()},
+           {Direction<3>::lower_eta()},
+           {Direction<3>::upper_eta()},
+           {Direction<3>::lower_zeta()},
+           {Direction<3>::upper_zeta()}}},
+      make_vector(make_coordinate_map<Frame::Logical, Frame::Inertial>(
+          affine_map_3d{affine_map{-1., 1., lower_bound[0], upper_bound[0]},
+                        affine_map{-1., 1., lower_bound[1], upper_bound[1]},
+                        affine_map{-1., 1., lower_bound[2], upper_bound[2]}})));
+}
+
+void test_periodic_brick(
+    const DomainCreators::Brick& brick,
```

Comment:
need to test other periodic boundaries as well (only periodic in x, periodic in x and y, etc)